js/app.js

```diff
@@ -12,4 +15,40 @@ export default {
   getSocketUrl() {
     return 'ws://localhost:8080/ws';
   },
+
+  followUnfollow(guid, type = 'follow') {
```

Comment:
I find having multiple files named the same thing but in different folders confusing.